pom.xml

```diff
@@ -290,6 +290,11 @@
       <version>${bdg-formats.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.bdgenomics.qc-metrics</groupId>
+      <artifactId>qc-metrics-core</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```

Comment:
Depending on a `SNAPSHOT` seems poor form, but I don't see the alternative (this code has been split off from core Adam but hasn't had a release yet).